doctors_surgery.html

```diff
@@ -26,17 +26,17 @@
     </nav>
 
     <label for="site-search">Search...</label>
-    <input type="search" name="search" aria-label="Search through site content">
+    <input type="search" id="site-search" name="search" aria-label="Search through site content">
     <button>Search</button>
 
     <section>
-        <h3><strong><em>COVID RESTRICTION UPDATE</em></strong></h3>
+        <h3><strong>COVID Restriction Update</strong></h3>
```

Comment:
I wondered whether the emphasis was needed on this title, given that it is already strong?